tools/clang/unittests/HLSLExec/LongVectors.cpp

```diff
@@ -32,20 +32,25 @@ template <typename T> const char *getHLSLTypeString() {
   static_assert(false && "Missing HLSL type string");
 }
 
-#define DATA_TYPE_NAME(TYPE, NAME, HLSL_STRING)                                \
+template <typename T> size_t getHLSLDataTypeSize() {
```

Comment:
I added this helper for some ops that I ended up removing. I kept it because I thought it might be helpful in the future. But now I'm thinking thats a waste. It's just bloat. Easy to add back in if it is needed.

Will remove.